src/components/Events/CardShort.jsx

```diff
@@ -4,10 +4,10 @@ import styled from "styled-components";
 function CardContent(props) {
   return (
     <StyleCardContent className="styleCardContent">
```

Comment:
With Styled Component the className attribute should become obsolete.

Reply:
Deleted them all. :-)